src/OpenTelemetry.AutoInstrumentation/InstrumentationDefinitions.cs

```diff
@@ -107,6 +107,11 @@ private static NativeCallTargetDefinition[] GetDefinitionsArray()
                 nativeCallTargetDefinitions.Add(new("System.ServiceModel", "System.ServiceModel.ChannelFactory", "InitializeEndpoint", new[] { "System.Void", "System.ServiceModel.Description.ServiceEndpoint" }, 4, 0, 0, 4, 65535, 65535, AssemblyFullName, "OpenTelemetry.AutoInstrumentation.Instrumentations.Wcf.WcfClientIntegration"));
                 nativeCallTargetDefinitions.Add(new("System.ServiceModel", "System.ServiceModel.ChannelFactory", "InitializeEndpoint", new[] { "System.Void", "System.ServiceModel.Channels.Binding", "System.ServiceModel.EndpointAddress" }, 4, 0, 0, 4, 65535, 65535, AssemblyFullName, "OpenTelemetry.AutoInstrumentation.Instrumentations.Wcf.WcfClientIntegration"));
             }
+
```

Comment:
nit. just to have only one instrumentation per condition.
```suggestion
#endif

#if NETFRAMEWORK
```